package.json

```
@@ -31,7 +31,7 @@
   ],
   "engines": {
     "npm": ">=8",
```

https://help.readymage.com/project-development/node.js-version

npm 10 with Node.js 18
Please do a NPM install after this change

I know I forgot this in https://github.com/aboutbits/zod-locales-formatjs/pull/9/files

```suggestion
    "npm": ">=10"
```